README.md

```diff
@@ -323,3 +329,15 @@ Although Asynction uses these fields to validate the input args of the callback
 | args       | [Schema Object](https://www.asyncapi.com/docs/specifications/2.0.0#schemaObject) | Schema of the arguments that are passed as input to the acknowledgement callback function. In the case of multiple arguments, use the `array` type to express the tuple. |
 
 In the future, the Message Ack Object may be extended with extra fields to enable additional documentation of the callback.
+
+### Security handers
+
+In order to support the [AuthN/AuthZ functionality](#security-authentication-and-authorization) of asynction, the [Security Scheme Object](https://www.asyncapi.com/docs/specifications/v2.2.0#securitySchemeObject) needs to be extended as follows:
+
+- A Security Scheme Object of `oauth2` type MAY include the `x-tokenInfoFunc` field.
```

Comment:
I think the `MAY` should be a `MUST` the only really optional one is the `x-scopeValidatefunc`

Reply:
The user will wind up getting and error when the security handler is created if the security function is missing.